master/internal/rm/resource_manager_iface.go

```diff
@@ -56,6 +56,7 @@ type ResourceManager interface {
 	// Kubernetes Namespaces.
 	DefaultNamespace(string) (*string, error)
 	VerifyNamespaceExists(string, string) error
+	RemoveEmptyNamespace(string, string) error
```

Comment:
*No change requested*

With how the code is structured, I think this is the right way to add `RemoveEmptyNamespace`. However, I do wish a kubernetes-specific function didn't need to be part of the resource manager interface.